stories/Uploader.stories.tsx

```diff
@@ -41,9 +41,16 @@ const Uploader = args => {
     const upupRef = useRef<UploadFilesRef>()
 
     const handleUpload = async () => {
-        await upupRef.current
-            ?.uploadFiles()
-            .then(data => console.log('data', data))
+        try {
+            const data = await upupRef.current?.uploadFiles()
+            if (data !== null) {
+                console.log('Upload successful')
+            } else {
+                console.log('Upload returned null.')
+            }
```

Comment:
@Elyyps you can replace this with the shorthand property 
```
console.log(`Upload ${ data ? 'successful' : 'returned null.'} `)
```
Otherwise great job !